tests/ossfuzz/fuzz_decode_alone.c

```diff
@@ -0,0 +1,41 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+/// \file       fuzz_decode_alone.c
+/// \brief      Fuzz test program for liblzma lzma_alone_decoder()
+//
+//  Author:     Maksym Vatsyk
+//
+//  Based on Lasse Collin's original fuzz.c fuzzer for liblzma
+//
+//  This file has been put into the public domain.
+//  You can do whatever you want with this file.
+//
+///////////////////////////////////////////////////////////////////////////////
+
+#include <inttypes.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include "lzma.h"
+#include "fuzz_common.h"
+
+
+extern int
+LLVMFuzzerTestOneInput(const uint8_t *inbuf, size_t inbuf_size)
+{
+	// Initialize a LZMA alone decoder using the above memory usage limit.
+	lzma_stream strm = LZMA_STREAM_INIT;
+	lzma_ret ret = lzma_alone_decoder(&strm, MEM_LIMIT);
+	if (ret != LZMA_OK) {
```

Comment:
Since `ret` isn't used outside of this if statement anymore, its enough to just say:

if (lzma_alone_decoder(&strm, MEM_LIMIT) != LZMA_OK)

and remove the `ret` variable from these programs.

Reply:
I apologize, this was a bad suggestion from me. I didn't realize the `ret` value was being used in the error message, which could be helpful if the fuzz targets somehow fail when creating the coders. Don't worry about reverting this, I will make the change after we merge.